### PR TITLE
feature LP132-missing-modifier-as-parameter

### PR DESCRIPTION
**Pull request by Lupillo17**
Add modifier as parameter to the component function.

### CODE REVIEW

**Review by carlosph816 — APPROVED** [review]
Well done!

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 1]
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
...
     messagePost: String,
-    sizeWidth: Float  = 1F,
+    sizeWidth: Float = 1F,
     onCommentClick: () -> Unit,
     onFavoriteClick: () -> Unit
-){
+) {
     Card(
         modifier = modifier
             .fillMaxWidth(sizeWidth)
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: This modifier property `fillMaxWidth(sizeWidth)` shouldn't be on this modifier, the function that implement this component is the only one that defines this kind of sizes. So you could remove this property

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 2]
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
...
-    ){
+    ) {
         Column(
             modifier = Modifier
                 .fillMaxWidth()
                 .padding(15.dp, 15.dp, 15.dp, 5.dp)
-        ){
+        ) {
             Row {
                 Column {
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: You can remove this column

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 3]
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
...
-                    Box(modifier = Modifier
-                        .fillMaxWidth(0.20f)
-                        .padding(0.dp)) {
+                    Box(
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: you can remove this box and just have the image and the column in this row.
In the column you can add in modifier the property padding start to have separation between the image and all the texts.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 4]
```diff
@@ -126,14 +128,14 @@ fun LpPostCard(
                     )
                 }
             }
-            Row(modifier = Modifier.padding(10.dp)){
+            Row(modifier = Modifier.padding(10.dp)) {
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt]: You can remove this `Row` and move this modifier to the text

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 5]
```diff
@@ -126,14 +128,14 @@ fun LpPostCard(
...
                     text = messagePost,
                     color = MaterialTheme.colorScheme.onPrimaryContainer,
                     fontSize = 15.sp
                 )
             }
-            Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.End){
+            Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.End) {
                 IconButton(onClick = onCommentClick) {
                     Icon(
                         Icons.Outlined.Comment,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:128]: Inside of the Icon, you can remove the parameter modifier as is not being used. This also apply for the modifier of the Icon favorite

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt [hunk 6]
```diff
@@ -167,15 +170,13 @@ fun LpPostCard(
...
+    modifier: Modifier = Modifier,
     icon: ImageVector,
     description: String,
     onCardClick: (String) -> Unit
 ) {
     Card(
-        modifier = Modifier
-            .height(120.dp)
-            .heightIn(120.dp, 250.dp)
-            .padding(8.dp),
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:178]: you can add padding property to the modifier `.padding(8.dp)`, you

File: app/src/main/java/com/iteneum/leasepert/MainActivity.kt [hunk 7]
```diff
@@ -4,13 +4,14 @@ import android.os.Bundle
...
 import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.example.ExampleView
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             LeasePertTheme {
-
+                ExampleView()
```
Comment (melissaquezada94) [inline app/src/main/java/com/iteneum/leasepert/MainActivity.kt]: If you are not going to use this function please delete it and the import as well

**Review by melissaquezada94 — APPROVED** [review]

**Pull request merged**